composer.json

```diff
@@ -16,52 +16,62 @@
         "source": "https://github.com/zfcampus/zf-apigility",
         "issues": "https://github.com/zfcampus/zf-apigility/issues"
     },
-    "repositories": [
-        { "type": "composer", "url": "https://packages.zendframework.com/" }
-    ],
     "extra": {
         "branch-alias": {
             "dev-master": "1.2-dev",
             "dev-develop": "1.3-dev"
+        },
+        "zf": {
+            "module": "ZF\\Apigility"
         }
     },
     "require": {
-        "php": ">=5.5",
-        "rwoverdijk/assetmanager": "~1.3",
-        "zendframework/zend-db": "~2.3",
-        "zendframework/zend-eventmanager": "~2.3",
-        "zendframework/zend-mvc": "~2.3",
-        "zendframework/zend-paginator": "~2.3",
-        "zendframework/zend-servicemanager": "~2.3",
-        "zendframework/zend-stdlib": "~2.3",
-        "zfcampus/zf-api-problem": "~1.0",
-        "zfcampus/zf-apigility-provider": "~1.0",
-        "zfcampus/zf-content-negotiation": "~1.0",
-        "zfcampus/zf-content-validation": "~1.0",
-        "zfcampus/zf-hal": "~1.0",
-        "zfcampus/zf-mvc-auth": "~1.0",
-        "zfcampus/zf-oauth2": "~1.0",
-        "zfcampus/zf-rest": "~1.0",
-        "zfcampus/zf-rpc": "~1.0",
-        "zfcampus/zf-versioning": "~1.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-db": "^2.8.1",
+        "zendframework/zend-eventmanager": "^2.6.3 || ^3.0.1",
+        "zendframework/zend-loader": "^2.5.1",
+        "zendframework/zend-mvc": "^2.7.9 || ^3.0.2",
+        "zendframework/zend-paginator": "^2.7",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1.1",
+        "zendframework/zend-stdlib": "^2.7.7 || ^3.0.1",
+        "zfcampus/zf-api-problem": "^1.2.1",
+        "zfcampus/zf-apigility-provider": "^1.2",
+        "zfcampus/zf-content-negotiation": "^1.2.1",
+        "zfcampus/zf-content-validation": "^1.3.3",
+        "zfcampus/zf-hal": "^1.4",
```

Comment:
@weierophinney `zf-hal` v1.4 can't be used with `zend-stdlib`.
Please have a look on https://github.com/zfcampus/zf-hal/pull/149